MultiMystery.py

```diff
@@ -159,8 +162,8 @@ def get_alive_threads():
             min_logical_seed = max_attempts
             from tqdm import tqdm
             with tqdm(concurrent.futures.as_completed(task_mapping.values()),
-                      total=len(task_mapping), unit="seed(s)",
-                      desc=f"Generating: {get_alive_threads()}") as progressbar:
+                      total=len(task_mapping), unit="seeds(s)", 
```

Comment:
This says seed(s) because it can be x seconds/seed
3 seconds per seeds doesn't make sense, hence this was the unit. 3 seconds per seeds(s) makes even less sense.

Reply:
Whoops, that one slipped through the cracks, from way back when.